apps/mobile-wallet/src/components/BiometricsWarningModal.tsx

```diff
@@ -29,7 +29,7 @@ const BiometricsWarningModal = memo<BiometricsWarningModalProps & ModalBaseProp>
   }
 
   return (
-    <BottomModal2 onDismiss={onDismiss} notScrollable modalId={id} title={`⚠️ ${t('Are you sure?')}`} noPadding>
+    <BottomModal2 notScrollable modalId={id} title={`⚠️ ${t('Are you sure?')}`} noPadding>
```

Comment:
Awesome, already so much cleaner 👏 
The context could also contain the `modalId` no? 
`BottomModal2` would then be super clean, no need to pass down props anymore. We may even get rid of the `ModalBaseProp` type I think?

Reply:
Let me have a look into it! Thanks for the feedback!